doc/classes/TileData.xml

```diff
@@ -70,15 +70,23 @@
 		<method name="get_navigation_polygon" qualifiers="const">
 			<return type="NavigationPolygon" />
 			<param index="0" name="layer_id" type="int" />
+			<param index="1" name="flip_h" type="bool" default="false" />
+			<param index="2" name="flip_v" type="bool" default="false" />
+			<param index="3" name="transpose" type="bool" default="false" />
 			<description>
 				Returns the navigation polygon of the tile for the TileSet navigation layer with index [param layer_id].
+				[param flip_h], [param flip_v] and [param transpose] allow transforming the returned polygon.
```

Comment:
```suggestion
				[param flip_h], [param flip_v], and [param transpose] allow transforming the returned polygon.
```
Oxford comma

Reply:
Hmm, is that a thing we do in the whole documentation? I don't mind changing it but I'd rather be sure the documentation team want's the oxford comma to be generally used.

Reply:
Inses, it's documented [here](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything)